lib/application/theme/memo_theme_colors.dart

```diff
@@ -46,6 +46,25 @@ MaterialColor buildClassicSecondarySwatch() {
   );
 }
 
+MaterialColor buildClassicNegativeSwatch() {
```

Comment:
I also like `danger` or `destructive`. Open to suggestions

Reply:
I do prefer destructive - totally personal preference, would be good with negative as well.

Danger is too specific

Reply:
Updated do destructive